draft-vvv-webtransport-quic.md

```diff
@@ -129,23 +131,70 @@ the client receives a TLS Finished message from the server.
 
 In order to identify itself as a WebTransport application, QuicTransport relies
 on TLS Application-Layer Protocol Negotiation {{!RFC7301}}.  The user agent MUST
-request the ALPN value of "wq" and it MUST NOT establish the session unless that
-value is accepted.
+request the ALPN value of "wq-draft01" and it MUST NOT establish the session
```

Comment:
I'm confused by this MUST NOT establish - if the server rejects the value it fails the handshake and closes the connection right?

Reply:
I changed it to "MUST close".  FWIW, this is just a restatement of quic-transport requirement.